### ISSUE TITLE
False positive from import/named when re-exporting.

### ISSUE THREAD

**Issue by jimbolla** [benmosher/eslint-plugin-import#251]
Given file `lodash.js` (showing 2 different ways to export):
```
import clone from 'lodash/fp/clone';
import filter from 'lodash/fp/filter';
export { default as flow } from 'lodash/fp/flow';
export { default as map } from 'lodash/fp/map';
export { default as sortBy } from 'lodash/fp/sortBy';

export { clone, filter };
```

and a usage in another file like this:
```
import { clone, filter, flow, map, sortBy } from './lodash.js';
```

eslint with `import/named` enabled with show false positives for `flow`, `map`, and `sortBy`.
```
  2:25  error  flow not found in '../lodash'    import/named
  2:31  error  map not found in '../lodash'     import/named
  2:36  error  sortBy not found in '../lodash'  import/named
```

The code executes fine, but the rule isn't detecting that syntax of export as valid.

**Comment by benmosher** [comment]
Interesting. [This test](https://github.com/benmosher/eslint-plugin-import/blob/master/tests/src/rules/named.js#L32) is for that case, though it tests "normal" named exports. 

I suspect something about re-exporting a `default` is causing the false positive. 

Will definitely investigate, thanks for the tip!
